src/Microsoft.ML.FastTree/FastTreeClassification.cs

```diff
@@ -277,7 +277,10 @@ protected override void InitializeTests()
             CalibratedModelParametersBase<FastTreeBinaryModelParameters, PlattCalibrator> model, Schema trainSchema)
             => new BinaryPredictionTransformer<CalibratedModelParametersBase<FastTreeBinaryModelParameters, PlattCalibrator>>(Host, model, trainSchema, FeatureColumn.Name);
 
-        public BinaryPredictionTransformer<CalibratedModelParametersBase<FastTreeBinaryModelParameters, PlattCalibrator>> Train(IDataView trainData, IDataView validationData = null)
+        /// <summary>
+        /// Trains a <see cref="FastTreeBinaryClassificationTrainer"/> using both training and validation data, returns a <see cref="ITransformer"/>.
```

Comment:
>ITransformer [](start = 133, length = 12)

As mentioned in other comments, it'd be more informative to specify the exact returned type here. Could you please also double-check other uses of `ITransformer` in those doc strings.

Reply:
Yes will search everywhere